signatures/rego/regosig/traceerego_test.go

```diff
@@ -6,10 +6,12 @@ import (
 	"os"
 	"testing"
 
+	"github.com/aquasecurity/tracee/signatures/rego/regosig"
```

Comment:
Are you using goimports to organise imports? I"m wondering why it's so different in terms of order and new lines added here and there?